### ISSUE TITLE
!say fails to parse @{target}.name's correctly.

### ISSUE THREAD

**Issue by Aquaricat** [UnicornHeartClub/Issues#107]
```
!target "Select target."
!say "@{target}.name's"
```
Obviously, the actual attribute is `@{target}.name`, but the parser is interpreting it as `@{target}.name's`.

**Comment by Aquaricat** [comment]
Actually, that example fails completely, with a
![image](https://user-images.githubusercontent.com/7004392/34910688-0e454c6c-f880-11e7-9d04-6cd3d72fbf59.png)
and this console log - [bug107-target-names01.log](https://github.com/UnicornHeartClub/Issues/files/1628809/bug107-target-names01.log)

**Comment by Aquaricat** [comment]
```
!target "Select target."
!say "@{target}.name's AC"
```
Fails silently (no output, no error notice), but provides this console log - [bug107-target-names02.log](https://github.com/UnicornHeartClub/Issues/files/1628810/bug107-target-names02.log)

**Comment by tlackemann** [comment]
This will be fixed in TTML v0.7.2

**Comment by Pjb518** [comment]
Working as expected. Closing! :tada:
